controllers/v2/entities.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * name : entities.js
+ * author : Aman Jung Karki
+ * created-date : 09-sep-2020
+ * Description : All entities v2 related information.
+ */
+
+
+/**
+ * dependencies
+ */
+
+const entitesV1 = require(ROOT_PATH + "/controllers/v1/entities");
+const entitiesHelper = require(MODULES_BASE_PATH + "/entities/helper.js");
+
+/**
+    * EntitiesV2
+    * @class
+*/
+
+module.exports = class EntitiesV2 extends entitesV1 {
+    /**
+     * @api {get} /kendra/api/v2/entities/listByEntityType/:entityType?page=:page&limit=:limit&search=:searchText 
+     * List of entities based on its type
+     * @apiVersion 2.0.0
+     * @apiGroup User
```

Comment:
@aman-tunerlabs - Please keep the same user group as v1 and as per the controller.